### PR TITLE
Fix Chrome multiple-headers bug for filenames containing commas

### PR DESCRIPTION
**Pull request by broxeph**
Chrome: ERR_RESPONSE_HEADERS_MULTIPLE_CONTENT_DISPOSITION

### CODE REVIEW

**Comment by tarkatronic** [comment]
@broxeph Thanks for submitting this PR! I'm glad to see this package is still in use!

I think it might be better, if it is possible, to escape the comma. Does this work as well if you change the replace from `.replace(',', '')` to `.replace(',', r'\,')`?

I would also love to see a test here to make sure this works as expected.

And it looks like I need to remove Python 3.3 needs to be removed from the Travis test grid in order for this to pass. In fact, the CI looks like it needs a general refresh. If you would like to simply remove the `- 3.3` reference from line 6 of `.travis.yml`, that should be sufficient for this PR to build, and then I can handle the refresh separately.

**Comment by broxeph** [comment]
My pleasure! Should be able to take a look on Tuesday.

File: excel_response/response.py [hunk 1]
```diff
@@ -58,17 +58,16 @@ def content(self, value):
...
             workbook = self.make_bytes(workbook.getvalue())
         else:
             self['Content-Type'] = 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'
-            self['Content-Disposition'] = 'attachment; filename={}.xlsx'.format(self.output_filename)
+            self['Content-Disposition'] = 'attachment; filename="{}".xlsx'.format(self.output_filename)
             workbook = save_virtual_workbook(workbook)
         self._container = [self.make_bytes(workbook)]
 
     def _serialize_list(self, data):
-        workbook = None
```
Comment (broxeph) [inline excel_response/response.py:71]: Unused (set later in either `if` or `else` block)

**Comment by broxeph** [comment]
Did a little more research and found that this is a Chrome "wontfix":

https://stackoverflow.com/questions/49003459/file-download-error-only-in-file-name-with-comma
https://answers.nuxeo.com/general/q/d8348e07fe5e441183bae07dfda00e40/Comma-in-file-name-cause-problem-in-Chrome-Browser

Rather than removing commas from file names, as in my original changeset, I changed it to SO's suggested solution and instead wrapped the `Content-Disposition` header in double quotes. Not sure how/whether you'd like to test this without adding Selenium or similar to the project :D

**Comment by broxeph** [comment]
For what it's worth, Python 3.4 is also EOL as of March 2019: https://www.python.org/downloads/release/python-3410/

**Comment by tarkatronic** [comment]
> For what it's worth, Python 3.4 is also EOL as of March 2019: https://www.python.org/downloads/release/python-3410/

As is Python 2.7, as of Jan 1, 2020. 😄 I will likely be removing them from the build matrix as soon as I have a chance to overhaul the CI.

As for tests, if there is no more substitution or escaping being done, then a new test is probably not necessary. Selenium seems a bit excessive for this case... :)

**Comment by tarkatronic** [comment]
Hey @broxeph I just got this released to PyPI; you should be able to install `2.0.4` now to get this fix. :-)

**Pull request merged**